### DISCUSSION TITLE
Document Announcement process

### DISCUSSION THREAD

**Discussion by smiller171** [music-assistant/support#617] [Q&A]
From the 'Get Loud' stream it seems like announcements are supposed to be a core feature of Music Assistant, however it seems quite unintuitive and/or buggy to get this functionality working.

I think it would make sense to clearly document how to use announcements, and if possible, improve the feature to make it more intuitive. (The `tts.*_say` services don't seem to work at all with resume?)

**Comment by marcelveldt** [comment]
Can you please create an issue report if you experience any issues instead of ranting about things being unintuitive and/or buggy ?
Note that this is a community driven software project, there will be bugs for sure. We can fix them if we receive good issue reports ;-)

**Reply by smiller171** [reply]
My intention wasn't to rant but to give feedback and have discussion on this. I didn't feel opening an issue was correct as I don't actually know if there is a bug in the software or between chair and keyboard. I seem to have missed the mark though. Please feel free to close this out if you don't feel it's appropriate for the discussion board.

**Comment by jazzyisj** [comment]
@smiller171 You need to send your tts service calls to a MASS media player.

https://github.com/music-assistant/hass-music-assistant/discussions/487

**Reply by smiller171** [reply]
yeah, I'm using the MASS media players. 🤷

**Comment by OzGav** [comment]
Relates to v1
